src/entities/courtinfo.entity.ts

```diff
@@ -0,0 +1,31 @@
+import { UserRole } from "src/common/enum";
+import { Column, Entity, Index, JoinColumn, ManyToOne, OneToMany, PrimaryGeneratedColumn } from "typeorm";
+import { Branch } from "./branch.entity";
+import {Comment} from "./comment.entity";
+@Entity()
+export class CourtInfo {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
+    
+    @Column()
+    name_of_court: string;
+
+    @Column()
+    description: string;
+
+    @Column()
+    id_branch: string;// khoá ngoại
+
+    @ManyToOne(()=>Branch, branch=> branch.courts)
+    @JoinColumn({name:'id_branch'})
+    branch: Branch
+
+    @Column({default:true})
+    is_active: boolean;
+
+    @Column({default:false})
+    is_locked: boolean;
```

Comment:
is_active, is_locked are also status, we can use `status` enum instead, status: [active, :deactivate, :lock]

Reply:
Thank you, I will update and create new migration.